cmd/clairctl/VERSION

```diff
@@ -0,0 +1 @@
+0.4.0
```

Comment:
It would be nice to version clair and clairctl together, that what we know that clairctl version X.X.X works with clair version X.X.X explicitly.